bot.py

```diff
@@ -112,16 +119,19 @@ def real_decorator(func):
         await client.change_presence(status=discord.Status.online, game=discord.Game(name=PLAYING_MESSAGE))
     starttime = datetime.now()
 
+
 @client.event
 async def on_resume():
     print("RESUMED")
     await log(1, "on_resume triggered!")
 
+
 @client.event
 async def on_message(message):
     if not starttime:
         return
-    if message.author.id in [client.user.id] + IGNORE_LIST or not client.get_server(WEREWOLF_SERVER).get_member(message.author.id):
+    if message.author.id in [client.user.id] + IGNORE_LIST or not client.get_server(WEREWOLF_SERVER).get_member(
+            message.author.id):
```

Comment:
uhh where did so many random newlines come from?

Reply:
All of the formatting edits (most unneeded) make it really hard to see what you actually changed